app/processing/schedule/schedule-pending-payment-schedules.js

```diff
@@ -11,8 +12,9 @@ const schedulePendingPaymentSchedules = async () => {
     const scheduledPaymentSchedules = await getScheduledPaymentSchedules(started, transaction)
     const validScheduledPaymentSchedules = getValidScheduled(scheduledPaymentSchedules)
     const updatedScheduledPaymentSchedules = await getUpdatedSchedule(validScheduledPaymentSchedules, started, transaction)
+    const removedDefunctPaymentSchedules = await getRemovedDefunctPaymentSchedules(updatedScheduledPaymentSchedules, started, transaction)
```

Comment:
is this returning the removed defucted payment schedules or is it returning the payment schedules with the defunct ones removed?

Reply:
Returning payment schedules with defunct ones removed